frontend/src/metabase/dashboard/actions/save.js

```diff
@@ -19,7 +19,7 @@ export const SAVE_DASHBOARD_AND_CARDS =
 
 export const saveDashboardAndCards = createThunkAction(
   SAVE_DASHBOARD_AND_CARDS,
-  function () {
+  function (preserveParameters = false) {
```

Comment:
Modify the function signature so that this accepts this flag as a parameter. This will make sure we could specify when we want to preserve parameter values when only making changes to certain dashboard attributes that don't have anything to do with the parameters themselves. e.g.
1. name
2. description
3. auto-apply filters
4. cache TTL